osu.Game.Rulesets.Osu/Difficulty/Skills/Flashlight.cs

```diff
@@ -45,11 +45,17 @@ private double strainValueOf(DifficultyHitObject current)
                 var osuPrevious = (OsuDifficultyHitObject)Previous[i];
                 var osuPreviousHitObject = (OsuHitObject)(osuPrevious.BaseObject);
 
+                OsuDifficultyHitObject osuLastPrevious;
```

Comment:
This naming is quite confusing. Maybe something like `osuAfterPrevious` or `osuNextPrevious`?
I'm not sure I like naming object we're currently iterating through `previous` either, but can't really come up with anything better.

Reply:
Something like `osuLoopCurrent` and `osuLoopNext` would be more descriptive

Reply:
`osuLoop-` sounds good.

Reply:
Resolved.